src/wp-includes/class-wp-theme-json.php

```diff
@@ -1899,7 +1899,7 @@ protected static function get_style_nodes( $theme_json, $selectors = array() ) {
 		 *
 		 * @param array $nodes Style nodes with metadata.
 		 */
-		return apply_filters( 'theme_json_get_style_nodes', $nodes );
```

Comment:
@scruffian @ajlende @draganescu I see that, in Gutenberg land, we name this filter `gutenberg_theme_json_style_nodes`. Do we need to take any action here? I haven't seen that we prefix the filters depending on environment (`wp_` or `gutenberg_`) in any other case. My understanding is that it's problematic for any consumer that want to use the filter. Though this rationale may have changed, so I've excluded this from https://github.com/WordPress/gutenberg/pull/44940 and can be looked at separately, if necessary.

Reply:
Thanks for flagging this @oandregal. I don't see any specific reason for using a different filter in the plugin vs core. I'll open a PR to match this.